lib/menu.rb

```diff
@@ -13,4 +23,10 @@ def view_menu
     MAIN_MENU
   end
 
+  def place_order(dish, quantity)
```

Comment:
I like how you have used the arguments in this method so that the user can input a quantity. I was trying to implement this in mine but could not find a way to link the use the quantity parameter to give me a total amount.